### DISCUSSION TITLE
delete user

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3452] [Q&A] [answered]
Hi ,
 I deleted user from nc_users_v2 table.  manually using sql query.
I am signing up the same user again using signup API.
But it showing     "msg": "User already exist"
How to resolve this?

Thanks and Regards,
Sumathy Venugopal

**Comment by o1lab** [comment, marked as answer]
Sorry, you will have to figure this out. We dont suggest to manually alter meta tables.
